Anlab.Mvc/Models/Order/OverwriteOrderResult.cs

```diff
@@ -15,13 +15,14 @@ public OverwriteOrderResult()
         }
         public IList<TestDetails> SelectedTests { get; set; }
         public IList<String> MissingCodes { get; set; }
+        public string ErrorMessage { get; set; }
         
         public string ClientId { get; set; }
         public int Quantity { get; set; }
         
         public decimal RushMultiplier { get; set; }
         public string LabworksSampleDisposition { get; set; }
 
-        public bool WasError => MissingCodes.Count > 0;
+        public bool WasError => MissingCodes.Count > 0 || ErrorMessage != null;

```

Comment:
@jSylvestre maybe better to use isnullOrWhitespace in there, since empty string error message shouldn't count as an error